### DISCUSSION TITLE
Firmware Update Info over MqTT

### DISCUSSION THREAD

**Discussion by Dartore** [tbnobody/OpenDTU#223] [Ideas]
Hi there, it‘s a nice feature in Info to see if a new update is ready!

Would it be possible to send this info over MqTT, if a new update is ready?

Br Dartore

**Comment by hubsi5** [comment]
Stimmt das wäre eine gute Idee.
Könnte ich mir dann gleich in meiner VIS anzeigen lassen, falls es ein Update gibt

**Comment by tbnobody** [comment]
Currently I don't like the idea that the ESP is fetching any  kind of version information from a foreign server. I would rather use the available resources to handle inverter stuff etc. There is also a lot of stuff which can go wrong. Just thinking of timeouts / non existing repositories etc. I don't trust the IP stack implementation enough to implement such a request.
